src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+    - `<provisioning_service_account_api-key>`: The API key of a service account with the Org Admininstrator role.
+
+    After your desired folder structure is in place, you could revoke the role from the organization and grant it on individual folders instead. A recommended security practice is to limit the users or service accounts with `FOLDER_ADMIN` at the level of the organization. When you [create a folder](#create-a-folder-and-grant-access-to-it), you can grant the `FOLDER_ADMIN` role at the level of the folder.
+
+1. To revoke the role at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request DELETE \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<user-id>/ORGANIZATION/<organization_id>/FOLDER_ADMIN' \
+           --header 'Authorization: <api-key>'
+    ~~~
+
+    Replace:
+    - `<user-id>`: the ID of the user or service account from whom to revoke the role.
+    - `<organization_id>`:your organization's ID.
+    - `<api-key>`: The API key of a service account with the Org Admininstrator role.
+
+1. Optionally, you can now delete the provisioning service account. In the remainder of this page, replace `<api-key>` with the API key for the folder-management service account.
+
+## Create a folder and grant access to it
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin)
+
+1. To create a folder at the organization level, set its `parent_id` field to `root`:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/folders' \
+           --header 'Authorization: Bearer <api-key>' \
+           --data '{"name" : "<folder_name>", "parent_id": "root"}'
+    ~~~
+
+    Replace:
+    - `<folder_name>`: a name for the folder.
+    - `<api-key>`: your service account's API key.
+
+    The response includes the new folder's ID.
+
+1. To create a descendent folder within a folder, set `parent_id` to the folder's ID:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/folders' \
+           --header 'Authorization: Bearer <api-key>' \
+           --data '{"name" : "<folder_name>", "parent_id": "<parent_id>"}'
+    ~~~
+
+    Replace:
+    - `<folder_name>`: a name for the folder.
+    - `<api-key>`: your service account's API key.
+    - `<parent-id>`: the ID of the folder that will contain the new folder.
+
+1. To grant your service account the `FOLDER_ADMIN` role on a folder and its descendent folders:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<service-account-id>/FOLDER/<folder-id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace:
```

Comment:
I would suggest the following here:
Replace:
- `<service-account-id>`: the service account ID (the role grantee)
- `<folder-id>` with the ID of the folder
- `<api-key>`: the role granter's API Key (not the API Key of the role grantee)

Reply:
Done